Tests/DefaultStringConvertibleTests/DefaultStringConvertibleTests.swift

```diff
@@ -82,40 +97,49 @@ final class DefaultStringConvertibleTests: XCTestCase {
     }
 
     func test_thatEnum_providesDeepDescription() {
-        let v = SomeEnum.Case3(value1: nil)
+        let v = SomeEnum.case3(value1: nil)
         let deepDescription = v.deepDescription
 
-        XCTAssertEqual(deepDescription, "SomeEnum.Case3(nil)")
+        XCTAssertEqual(deepDescription, "SomeEnum.case3(nil)")
         print("\n", deepDescription, "\n")
 
     }
 
     func test_thatClass_providesDeepDescription() {
+
+	// Since in Swift Foundation customMirror for NSDictionary is not yet implemented
+	#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
         let v = SomeClass(prop1: [1, 2, 3], prop2: [0, "hello world", NSDictionary()])
-        let deepDescription = v.deepDescription
+	#else
+        let v = SomeClass(prop1: [1, 2, 3], prop2: [0, "hello world", [String : String]()])
```

Comment:
this complexity isn't necessary. let's just have the `let v = ...` in the else block